packages/jobs/lib/app.ts

```diff
@@ -25,6 +26,7 @@ try {
     cronAutoIdleDemo();
     deleteOldActivityLogs();
     deleteSyncsData();
+    reconcileTemporalSchedules();
```

Comment:
Can we make this optional because temporal does not exist in oauth install

Reply:
This is running on `jobs` which assumes that temporal is running.